config/carbonapi.yaml

```diff
@@ -136,3 +136,11 @@ monitoring:
         start: 0.0
         bucketsNum: 40
         bucketSize: 0.05
+    findDurationExpHistogram:
+        start: 0.06
+        bucketsNum: 30
+        bucketSize: 3
```

Comment:
Multiples of 3 are usually not very comfortable. Are there any particular reasons to use multiples of 3?